vane/nrfu_client.py

```diff
@@ -89,6 +93,17 @@ def setup(self):
         # Run Vane with the generated duts and definitions file
         print("\x1b[32mStarting Execution of NRFU tests via Vane\x1b[0m")
 
+    def ask_report_detail(self):
+        """Ask user if summary or detailed report
+        Puts the user's choice in self._report_detail_level
+        """
+        self._report_detail_level = True  # default set to generate only summary report
+        user_choice = input(
+            "Do you want detailed report?(Press Enter for the default option: "No") [y/yes/n/no]:"
```

Comment:
Minor changes to prompt phrasing and spacing:
```suggestion
            "Do you want a detailed report? (Press Enter for the default option: "No") [y/yes/n/no]: "
```